drivers/hmis/spec/requests/hmis/update_relationship_to_ho_h_spec.rb

```diff
@@ -86,9 +86,9 @@ def perform_mutation
 
     context 'when new HoH entered before move-in' do
       it 'should transfer move-in date to new HoH' do
-        expect { perform_mutation }.to change { hoh.relationship_to_ho_h }.from(1).to(99).
+        expect { perform_mutation }.to change { hoh.relationship_to_ho_h }.from(1).to(3).
           and change { hoh.move_in_date }.from(hoh_move_in_date).to(nil).
-          and change { hhm.relationship_to_ho_h }.from(2).to(1).
+          and change { hhm.relationship_to_ho_h }.from(3).to(1).
```

Comment:
I'm not seeing a test for 2 -> 4.  Feels like maybe we should have one.  (Changed a child to the HoH, previous HoH should be (4) "Head of household’s other relation member"

Reply:
Added!